src/shared/Loader.tsx

```diff
@@ -2,7 +2,7 @@ import { CircularProgress } from "@mui/material";
 
 function Loader() {
   return (
-    <div className="flex justify-center items-center w-full h-screen">
+    <div className="flex justify-center items-center w-full ">
```

Comment:
Why remove the h-screen here. 

The h-screen was added to ensure the loader is in the middle of the page. Just call the circular progres in the place you wanna use Loader

Reply:
Its been fixed. I will raise a new PR